pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -60,16 +60,25 @@ import (
 	"github.com/karmada-io/karmada/pkg/util/restmapper"
 )
 
+// well-know labels
 const (
-	// bindingDependedIDLabelKey is the resource id of the independent binding which the attached binding depends on.
-	bindingDependedIDLabelKey = "resourcebinding.karmada.io/depended-id"
-
-	// bindingDependedByLabelKeyPrefix is the prefix to a label key specifying an attached binding referred by which independent binding.
-	// the key is in the label of an attached binding which should be unique, because resource like secret can be referred by multiple deployments.
-	bindingDependedByLabelKeyPrefix = "resourcebinding.karmada.io/depended-by-"
-	// bindingDependenciesAnnotationKey represents the key of dependencies data (json serialized)
-	// in the annotations of an independent binding.
-	bindingDependenciesAnnotationKey = "resourcebinding.karmada.io/dependencies"
+	// dependedIDLabelKey is added to the attached binding, it describes the
+	// resource id of the independent binding which the attached binding depends on.
+	dependedIDLabelKey = "resourcebinding.karmada.io/depended-id"
```

Comment:
I've been confused whether we plan to use this label to replace `resourcebinding.karmada.io/depended-by-`.

Reply:
Yes, that's exactly what I'm going to do. The label will be removed.

Reply:
thanks, it makes sense. I can close this [issue](https://github.com/karmada-io/karmada/issues/4526) now :dash:.

Reply:
I think this will also be a prefix, like this:
`resourcebinding.karmada.io/depended-id-xxx`

Let's talk about it in the next PR.